src/releng_docs/services_shipit.rst

```diff
@@ -116,10 +116,16 @@ Initially we will integrate the legacy ship-it/release-runner infrastructure wit
 7. The pipeline service sees that S is finished, and creates step P via the Taskcluster service.
 
 
-Steps
------
+Step Services
+-------------
 
-Step Services API
+.. toctree::
+    :maxdepth 1
+
+    shipit_signoffs/shipit-signoffs
+
+
```

Comment:
Yeah, I thought so too. I tried making it not have any depth (so we'd only have app names in the TOC but that didn't seem to work.